arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-custom-gas-token-chain.md

```diff
@@ -0,0 +1,124 @@
+| `DataAvailabilityCommittee` |  `boolean`   |  Should be set to `true` since only Anytrust chains can accept `ERC-20` tokens.                                                              |
+
+
+### 3. Token Approval Before Deployment Process
+
+In Custom gas token Orbit chains, the owner needs to give allowance to the `rollupCreator` contract before starting the deployment process so that `RollupCreator` can spend enough tokens for the deployment process. For this purpose, we defined two APIs on the Orbit SDK:
+
+#### A. createRollupEnoughCustomFeeTokenAllowance
+   
+This API gets related inputs and checks if the `rollupCreator` contract has enough Allowance on the token from the owner:
+   
+```js
+import {createRollupEnoughCustomFeeTokenAllowance} from '@arbitrum/orbit-sdk';
+
+const allowanceParams = {
+nativeToken,
+account: deployer.address,
+publicClient: parentChainPublicClient,
+};
+
+const enough Allowance = createRollupEnoughCustomFeeTokenAllowance(allowanceParams)
+```
+
+To build the `allowanceParams` object as shown in the example above, you need to provide with the following:
+
+| Parameter                   | Type            |  Description                                                                                                                               |
+|-----------------------------|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| `nativeToken`               |  `Address`      |  The contract address of the `ERC-20` token your chain will use for `gas` fees.                                                              |
+| `account`                   |  `Address`      |  The  address  Orbit chain's|
+| `publicClient`              |  `PublicClient` |  The `PublicClient` object [as defined by the Viem library](https://viem.sh/docs/clients/public.html).                                       |
+
+#### B. createRollupPrepareCustomFeeTokenApprovalTransactionRequest
+   
+This API gets related inputs and creates the transaction request to secure enough Allowance from the owner to the `RollupCreator` to spend `nativeToken` on the deployment process.
+
+Example:
+
+```js
+import {createRollupEnoughCustomFeeTokenAllowance} from '@arbitrum/orbit-sdk';
+
+const allowanceParams = {
+nativeToken,
+account: deployer.address,
+publicClient: parentChainPublicClient,
+};
+
+const approvalTxRequest = await createRollupPrepareCustomFeeTokenApprovalTransactionRequest(
+    allowanceParams,
+);
+```
+### 4. Deployment Process
+
+The overall deployment process, including the use of APIs like `createRollupPrepareConfig` and `createRollupPrepareTransactionRequest`, remains similar to the [Rollup deployment](orbit-sdk-deploying-rollup-chain.md) process. However, attention must be given to incorporating the `ERC-20` token details into these configurations.
```

Comment:
Shouldn't we point at the AnyTrust doc better? (since this is going to be an AnyTrust chain)